bootstrap/bootstrap.php

```diff
@@ -251,6 +251,8 @@
 /** @var \craft\web\Application|craft\console\Application $app */
 $app = Craft::createObject($config);
 
+Craft::bootstrapContainer();
+
```

Comment:
This needs to be _after_ we initialize `$app`, as the config we are using for DI may reference Craft's config component.